x-pack/plugins/apm/public/tutorial/config_agent/opentelemetry_instructions.tsx

```diff
@@ -75,9 +77,19 @@ export function OpenTelemetryInstructions({
         }
       ),
       render: (_, { value }) => (
-        <EuiText size="s" color="accent">
-          {value}
-        </EuiText>
+        <>
+          <EuiText size="s" color="accent">
+            {value}
+          </EuiText>
+          {value && (
+            <EuiButtonIcon
+              aria-label="Copy to clipboard"
```

Comment:
Do we need `i18n.translate` for aria labels?

Reply:
Yes, good catch, i will add it now